docs/source/index.rst

```diff
@@ -49,27 +49,39 @@ Quick Start
 -----------
 
 The fastest way to get up and running with DeepChem is to run it on
-Google Colab. Check out one of the `DeepChem Tutorials`_ or this
-`forum post`_ for Colab quick start guides.
+Google Colab. Check out one of the `DeepChem Tutorials`_.
 
-If you'd like to install DeepChem locally,
-we recommend installing deepchem which is nightly version and RDKit.
-RDKit is a soft requirement package, but many useful methods depend on it.
+on **Google Colab**
 
 .. code-block:: bash
 
-    pip install tensorflow~=2.4
-    pip install --pre deepchem
-    conda install -y -c conda-forge rdkit
+    !pip install --pre deepchem
 
-Then open your python and try running.
+On **Windows/Linux**
+
+.. code-block:: bash
+
+    pip install deepchem[tensorflow]
```

Comment:
For quickstart, users can start by installing deepchem as `pip install deepchem`. This section is probably not needed.